app/Http/Controllers/ConferencesController.php

```diff
@@ -111,14 +112,25 @@ public function show($id)
             return redirect('/');
         }
 
-        $talksAtConference = $conference->myTalks()->map(function ($talkRevision) {
-            return $talkRevision->talk->id;
+        $talks = auth()->user()->talks->map(function ($talk) use ($conference) {
+            /** @var Talk $currentTalk */
+            $currentTalk = $talk->current();
+            $submission = $talk->getSubmission($conference);
+            $acceptance = $submission->acceptance;
+            return [
+                'id' => $talk->id,
+                'title' => $currentTalk->title,
+                'url' => $currentTalk->getUrl(),
+                'submitted' => $acceptance ? false : !!$submission,
```

Comment:
Hm, I think of it as submitted even if it's accepted--can we pass it that way and then update the template to still do what you intend?

Reply:
I can update the template. 👍